hazelcast/include/hazelcast/client/InitialMembershipEvent.h

```diff
@@ -44,12 +46,10 @@ namespace hazelcast {
          */
         class HAZELCAST_API InitialMembershipEvent {
         public:
-
-            /**
-             * Constructor
-             */
             InitialMembershipEvent(Cluster &cluster, const std::vector<Member> &members);
```

Comment:
Is this constructor still needed ? I could not find any usages.

Reply:
Removed.